rac/rac.go

```diff
@@ -71,6 +72,12 @@ type RacPartInfo struct {
 	Value   float64
 }
 
+type RacNFTInfo struct {
+	Creator            string
+	ContentHash        string
+	ContentDescription string
+}
+
```

Comment:
Since the block struct for NFT is similar to that of Smart Contracts, we can remove this struct